man/manual.md

```diff
@@ -357,9 +357,15 @@ To suppress the error message, a comment can be added:
 
     void f() {
         char arr[5];
+        arr[10] = 0;    // cppcheck-suppress[arrayIndexOutOfBounds]
+    }
 
-        // cppcheck-suppress arrayIndexOutOfBounds
```

Comment:
IMHO if the old inline suppression still works it would be useful to have it still in the documentation.

Reply:
I have put it in line 389, but just a litter description for it.